mava/adders/reverb/base.py

```diff
@@ -74,6 +89,17 @@ def spec_like_to_tensor_spec(
     return tf.TensorSpec.from_spec(spec, name="/".join(str(p) for p in paths))
 
 
+def get_trans_net_agents(
+    trajectory: Union[Trajectory, mava_types.Transition], entry_net_keys: Dict[str, str]
+) -> Tuple[List, Dict[str, List]]:
+    agents = sort_str_num(trajectory.actions.keys())
+    unique_nets = sort_str_num(set(entry_net_keys.values()))
+    trans_nets_agent: Dict[str, List] = {key: [] for key in unique_nets}
```

Comment:
`trans_nets_agent` ? I think docstrings for this function would help a bit.

Reply:
Thanks. Yay, I really need to update the variable names a bit. I updated this variable name and added a docstring :+1: